.travis.yml

```diff
@@ -29,13 +29,16 @@ after_success:
 stages:
 - name: test
 - name: release
-  if: (branch = master AND type = push) OR (tag IS present)
+  if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork
```

Comment:
Is this going to affect us or do we not use travis?

Reply:
No, it's not going to affect us - we have a separate circleci pipeline [here](https://github.com/ovotech/scala-steward/blob/master/.circleci/config.yml).